src/vm-repair/HISTORY.rst

```diff
@@ -2,10 +2,15 @@
 Release History
 ===============
 
+2.0.4
+++++++
+Added new parameter `os-disk-type` to `vm repair create` to let users specify the repair vm's os disk storage account type. 
+
```

Comment:
Adding new parameter is a feature for vm-repair and therefore new version should be `2.1.0`.

For more info about extension versioning, please refer to the doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table